TOMP-API.yaml

```diff
@@ -63,27 +63,34 @@ paths:
       - $ref: "#/components/parameters/apiVersion"
     post:
       description:
-        Returns available transport options for given coordinates and radius. <p>Start time can be defined, but is optional. If startTime is not provided, but required by the third party API, a default value of "Date.now()" is used. [from MaaS-API /listing].
-        During the routing phase this service can be used to check availability without any state changes. <p>In the final check, just before presenting the alternatives to the user, a call should be made using `provideIds`, requesting the TO to provide unique IDs to reference to during communication with the MP.
+        Returns planning options for the given travel plan. <p>Start time can be defined, but is optional. If startTime is not provided, but required by the third party API, a default value of "Date.now()" is used. [from MaaS-API /listing].
+        During the routing phase this service can be used to check availability without any state changes. <p>In the final check, just before presenting the alternatives to the user, a call should be made using `bookable`, requesting the TO to provide booking IDs to reference to during communication with the MP.
         <p>see (2.1) in the process flow - planning
       tags:
         - planning
         - TO
+      parameters:
+        - name: bookable
+          in: query
+          description: Specifies whether bookable IDs should be returned with the results
+          schema:
+            type: boolean
+            default: false
       requestBody:
         content:
           application/json:
             schema:
-              $ref: "#/components/schemas/planningCheck"
+              $ref: "#/components/schemas/travelPlan"
       responses:
         "201":
           description: Available transport methods matching the given query parameters. If no transport methods are available, an empty array is returned.
           content:
             application/json:
               schema:
-                $ref: "#/components/schemas/planningOptions"
+                $ref: "#/components/schemas/planningOption"
```

Comment:
ha ha. It's actually not an option. "PlanningResult"? The result contains options.

Reply:
True, but the URL is /planning-options/ so the objects in it should be planning-option to match the expectation people would have from it being a RESTful API. Would you be open to changing the URL to something like /plannings/ and returning a 'planning' or something like that?